src/types.jl

```diff
@@ -104,3 +103,10 @@ macro NestedVector(T::Symbol, N::Int)
     push!(cur_code.args, Symbol(T))
     esc(code)
 end
+
+
+######################
+# Enums for sentiment datasets
+
+@enum setMap train = 1 test = 2
```

Comment:
Sorry for the mixed messages I think using strings or symbols is fine for this split, or at least better than enum.
They are very single use just for saying which to load, right,?

Reply:
So should I keep `train` and `test` with `pos`, `neg` and `neu` as strings.